pbp.sh

```diff
@@ -1,27 +1,14 @@
-echo "$(tput setaf 5)Pulling and bundling consumer-layouts...$(tput sgr0)"
-cd consumer-layouts
-git pull
-bundle install
+#!/usr/bin/env bash
```

Comment:
It's called a "shebang" for the Hash Bang at the front, and POSIX understands it to mean what interpreter to use. The classic way to do it would be `#!/bin/bash` but it's become good practice in the recent years to use the `/usr/bin/env` binary to account for cases where /bin/bash is not the right location.